### PR TITLE
Get ready for release 7.0.0

### PR DESCRIPTION
**Pull request by rocky**
Go over release notes. Add a way to correct spelling mistakes in ChangeLog produced (which is used to create CHANGES.rst)

### CODE REVIEW

File: ChangeLog-spell-corrected.diff [hunk 1]
```diff
@@ -0,0 +1,3526 @@
+--- ChangeLog	2024-08-09 10:10:09.519394315 -0400
++++ ChangeLog-spell-corrected	2024-08-09 10:08:54.403769916 -0400
```
Comment (mmatera) [inline ChangeLog-spell-corrected.diff:2]: I don't understand why we want to versioning this `diff` file.

**Review by mmatera — APPROVED** [review]
LGTM

**Comment by mmatera** [comment]
> Go over release notes. Add a way to correct spelling mistakes in ChangeLog produced (which is used to create CHANGES.rst)

Would not be enough to pick the ChangeLog since the last release and keep CHANGES.rst clean of typos?

**Comment by rocky** [comment]
> > Go over release notes. Add a way to correct spelling mistakes in ChangeLog produced (which is used to create CHANGES.rst)
> 
> Would not be enough to pick the ChangeLog since the last release and keep CHANGES.rst clean of typos?

The ChangeLog goes into the distribution. It provide a slightly more detailed and slightly cleaned up version of the actual commit changes.

**Pull request merged**